tests/ctest-requirements.txt

```diff
@@ -1,5 +1,5 @@
 cmake==3.25.2  # Configure/build C++ tests
-conan==1.59  # Install dependencies for C++ tests.
+conan==1.60.1  # Install dependencies for C++ tests.
```

Comment:
[plan] This conflict only occurs due to tests, so it's a real shame it limits us in our production code. 

I think the correct thing to do is isolate the CMake tests somehow.  It is already a bit odd the way we build and run a CMake project from within a pytest test case.  But that's obviously not a small fix.

Reply:
Added https://github.com/OpenAssetIO/OpenAssetIO-TraitGen/issues/32

Reply:
Perfect, thanks.